src/services/gta_data/yim_fipackfile.cpp

```
@@ -84,16 +84,58 @@ namespace big
 		return output;
 	}
 
+	static std::filesystem::path get_game_folder_path()
+	{
+		std::wstring game_module_path(MAX_PATH, '\0');
+
+		if (GetModuleFileNameW(nullptr, game_module_path.data(), game_module_path.size()))
```

When I used to manually map my cheat, I had problems with using the WinAPI to retrieve information about GTA's process. Has this been tested with MMing? I don't know if the project only supports LLA injection or not, but that might be something to consider.

---

I can't even MM with current nightly (without this PR merged in) so ¯\_(ツ)_/¯

---

Was the user having problems injecting with manual mapping?

---

No, i'm just suspecting std::filesystem::current_path being able to return bogus result on some systems, so i'm just removing one extra layer of potential non controlled errors